planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/heuristics/BlindSearch.java

```diff
@@ -33,13 +37,12 @@ public BlindSearch() {
    public BlindSearch(int maxIter) {
       super(maxIter);
    }
-   
-   
-   
+
    @Override
-   public Solution[] computeOptimizationProblemForAllDifferentSolutions(SuitableOptions cloudOffers, QualityInformation requirements,
-         Topology topology, int numPlansToGenerate) {
-      return super.filterUniqueSolutions(computeOptimizationProblem(cloudOffers,requirements,topology,numPlansToGenerate));
+   public Solution[] computeOptimizationProblemForAllDifferentSolutions(SuitableOptions cloudOffers,
+         QualityInformation requirements, Topology topology, int numPlansToGenerate) {
+      return super.filterUniqueSolutions(
```

Comment:
Thanks @kiuby88 . It is not necessary. I'll remove it. I wrote it only for clarity (to myself) about method location.